### ISSUE TITLE
Is there a way to track created processes?

### ISSUE THREAD

**Issue by Ou7law007** [hasherezade/tiny_tracer#11]
My sample calls CreateProcessInternalW to create a new process, is there a way track it without attaching a debugger or hooking the function?

**Comment by hasherezade** [comment]
Sorry but currently TinyTracer does not provide following children processes. However, as far as I know Pin gives such option, so I will check if I can add it.

**Comment by Ou7law007** [comment]
Thank you.
I was able to run the child process on its own but it requires specific arguments. Is there a way to maybe pass arguments to tiny tracer?

**Comment by hasherezade** [comment]
> I was able to run the child process on its own but it requires specific arguments. Is there a way to maybe pass arguments to tiny tracer?

Yes. You just need to add the parameter in the `run_me.bat`, just after the `"%TARGET_APP%"`
https://github.com/hasherezade/tiny_tracer/blob/a38128de2aba8996ce84cc394370ded5987700bb/install32_64/run_me.bat#L79

**Comment by Ou7law007** [comment]
Awesome. Thank you

**Comment by cecio** [comment]
hey @hasherezade , hope you are fine!

I made a small modification to add the possibility (controlled by an option in .INI file) to follow subprocesses.
If you are interested, I can open a pull request, so we can discuss the implementation and any rework you may think it's needed.
Thanks!
